File: backend/models/Temple.js
const mongoose = require('mongoose');

const templeSchema = new mongoose.Schema({
  name: {
    type: String,
    required: true
  },
  location: {
    type: String,
    required: true
  },
  description: {
    type: String,
    required: true
  },
  shortDescription: {
    type: String,
    required: true
  },
  mainImage: {
    type: String,
    required: true
  },
  images: [String],
  history: {
    type: String,
    required: true
  },
  significance: {
    type: String,
    required: true
  },
  createdAt: {
    type: Date,
    default: Date.now
  }
});

module.exports = mongoose.model('Temple', templeSchema);

File: backend/ParasnathAPI/Models/Temple.cs
using System;
using System.Collections.Generic;

namespace ParasnathAPI.Models
{
    public class Temple
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Location { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public string ShortDescription { get; set; } = string.Empty;
        public string MainImage { get; set; } = string.Empty;
        public string History { get; set; } = string.Empty;
        public string Significance { get; set; } = string.Empty;
        public string Architecture { get; set; } = string.Empty;
        public string BestTimeToVisit { get; set; } = string.Empty;
        public string Facilities { get; set; } = string.Empty;
        public List<string> Images { get; set; } = new();
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    }
}
